src/util/HDWalletProvider.js

```diff
@@ -1,54 +1,64 @@
-const TruffleHDWalletProvider = require('truffle-hdwallet-provider')
-const Web3 = require('web3')
+const TruffleHDWalletProvider = require("@truffle/hdwallet-provider");
+const Web3 = require("web3");
```

Comment:
Do you really want to commit all these format changes and be blamed on those lines? Maybe you can disable your format on save for this repo?